frontend/appflowy_flutter/lib/user/presentation/screens/sign_in_screen/widgets/third_party_sign_in_buttons.dart

```diff
@@ -0,0 +1,168 @@
+import 'package:appflowy/core/config/kv.dart';
+import 'package:appflowy/core/config/kv_keys.dart';
+import 'package:appflowy/generated/flowy_svgs.g.dart';
+import 'package:appflowy/generated/locale_keys.g.dart';
+import 'package:appflowy/startup/startup.dart';
+import 'package:appflowy/user/application/sign_in_bloc.dart';
+import 'package:easy_localization/easy_localization.dart';
+import 'package:flowy_infra/size.dart';
+import 'package:flowy_infra_ui/flowy_infra_ui.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+class ThirdPartySignInButtons extends StatelessWidget {
+  final bool isMobile;
+  final Alignment contentAlignment;
+
+  /// Used in DesktopSignInScreen and MobileSignInScreen
+  const ThirdPartySignInButtons({
+    Key? key,
+    required this.isMobile,
+    this.contentAlignment = Alignment.center,
+  }) : super(key: key);
```

Comment:
use super.key instead.

Reply:
Why do we need to change it?

Reply:
The new syntax is much shorter and cleaner.